backend/viewModule/views.py

```diff
@@ -262,24 +253,6 @@ def all_state_total_releases(request):
     return HttpResponse(response, content_type='application/json')
 
 
-""" 
```

Comment:
@adwait97 This was here because there is no documentation on request/response format.